src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const body = document.body;
+
+async function handlePromise(promise) {
+  try {
+    const message = await promise;
+
+    body.insertAdjacentHTML(
+      'beforeend',
+      `
+      <div data-qa="notification" class="success">
+        ${message}
+      </div>
+    `,
+    );
+  } catch (errorMessage) {
+    body.insertAdjacentHTML(
+      'beforeend',
+      `
+      <div data-qa="notification" class="error">
+        ${errorMessage}
+      </div>
+    `,
+    );
+  }
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
```

Comment:
The error message here is an Error object. When this is caught in the catch block, it will be converted to a string like 'Error: First promise was rejected'. If you want to display just the message, consider using errorMessage.message in the catch block.